code/style.css

```diff
@@ -1,7 +1,657 @@
+
+  nav a:hover {
+    color: orange;
+  }
+
+  .button-header {
+    display: block;
+    width: 120px;
+    height: 35px;
+    padding: 10px;
+    background-color: #033150;
+    border: none;
+    color: white;
+    text-transform: uppercase;
+    font-family: 'Poppins', Sans-Serif;
+    font-size: 12px;
+    font-weight: bolder;
+    cursor: pointer;
+    outline: none;
+    border-radius: 4px;
+  }
+
+  .button-header:hover {
+    background-color: #043d64;
+  }
+
+  h1.header-title {
+    align-items: center;
+    padding: 20px;
+    font-family: 'Abril Fatface', Cursive;
+    font-weight: lighter;
+    font-size: 80px;
+    color: orange;
+  }
+
+  h2.header-subtitle {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    font-size: 30px;
+    color: white;
+    font-family: 'Poppins';
+    font-weight: lighter;
+  }
+}
+
+
+/* FORM SECTION */
+
+section.form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 40px;
+  align-items: center;
+}
+
+.form-textbox {
+  background-color: #033150;
+  color: white;
+  width: 80%;
+  padding: 30px;
+  margin: 30px 0px;
+}
+
+.form-textbox-header {
+  font-family: 'Abril Fatface', Cursive;
+  font-size: 30px;
+  font-weight: normal;
+  margin: 0 0 10px 0;
+}
+
+.form-textbox-text {
+  margin-top: 0px;
+  font-family: 'Poppins', Sans-Serif;
+  font-weight: lighter;
+  font-size: 14px;
+  text-align: justify;
+}
+
+form {
+  width: 80%;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  background-color: white;
+  color: #033150;
+  font-family: 'Poppins', Sans-Serif;
+  padding: 30px;
+}
+
+.form-select-group {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.form-size-group {
+  width: 50%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+  margin-bottom: 5px;
+}
+
+.form-input-radio {
+  width: 30px;
+  height: 20px;
+  margin: 0 5px 0 0;
+}
+
+.form-size-group .form-label-input {
+  font-size: 14px;
+}
+
+.form-dates-group {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 10px;
+}
+
+.form-dates-checkbox-group {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin: 5px 0;
+}
+
+.form-dates-checkbox-group .form-label-input {
+  font-size: 14px;
+}
+
+.form-input-checkbox {
+  width: 18px;
+  height: 18px;
+}
+
+.form-extra-group {
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+}
+
+.form-label-heading {
+  width: 100%;
+  margin: 10px 0 10px 0;
+  font-size: 18px;
+  font-weight: bold;
+  text-transform: uppercase;
+}
+
+.form-label-input {
+  font-size: 12px;
+  color: black;
+}
+
+.form-select-box, .form-input-date, .form-input-text {
+  width: 100%;
+  height: 30px;
+  border: none;
+  background-color: #f5f5f5;
+  font-size: 14px;
+  font-family: 'Poppins', Sans-Serif;
+  color: #696969;
+}
+
+.form-select-box:focus, .form-input-date:focus, .form-input-text:focus {
+  outline: none;
```

Comment:
Be aware that the outline is there for accessibility reasons. We will talk more about accessibility so no worries :)